lib/core/src/lib/viewer/viewer.module.ts

```diff
@@ -49,7 +49,7 @@ import { A11yModule } from '@angular/cdk/a11y';
         ExtensionsModule,
         CommonModule,
         MaterialModule,
-        TranslateModule.forChild(),
```

Comment:
this should be a problem for lazy loading

Reply:
Put it back